module/data/item/templates/ability.mjs

```diff
@@ -58,6 +58,28 @@ export default class AbilityTemplate extends SystemDataModel {
                 integer: true,
                 label: "ED.Item.Ability.strain"
             } ),
+            difficulty: new foundry.data.fields.SchemaField( {
```

Comment:
Are there any other (edge) cases? E.g. like for spells where it's either a fixed difficulty OR the defense of the target whichever is higher/lower etc?
Also, how do we know which field to take if some reason more than one has a value?
We might need to rethink this...